docs/content/configuration/indexing-service.md

```diff
@@ -192,7 +192,7 @@ An affinity config can be provided.
 |Property|Description|Default|
 |--------|-----------|-------|
 |`type`|`fillCapacityWithAffinity`.|fillCapacityWithAffinity|
-|`affinity`|A map to String to list of String host names.|{}|
+|`affinity`|JSON object mapping a datasource String name to a list of indexing service middle manager host:port String values.|{}|
```

Comment:
It might be worth noting that the host name should match what is configured on the middle manager. Druid doesn't perform dns resolution so it will not match if the host names are not exactly what the middle managers are announcing themselves as.